### DISCUSSION TITLE
How to include AppleMail in browserslist

### DISCUSSION THREAD

**Discussion by ArthurGoupil** [browserslist/browserslist#825] [General]
Hello!

I'm currently integrating a browserslist and it appears we consider some browsers deprecated because they don't explicitly mention a browser: ex `Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko)`

After some research it may be AppleMail < v6 that does not mention any browser (see [here](http://m.webapps-online.com/online-tools/user-agent-strings/dv/browser52900/apple-mail?)). MacOS 10.15.7 has been released on sept 2021 so it's quite a shame to consider them as deprecated.

Is there a way to integrate this in our browserslist without being too permissive in the meantime?

Thanks a lot

**Comment by ai** [comment]
Could it be some old Safari? We do not support `AppleMail` (and honestly don’t want to support it), but you can write something like:

```
# AppleMail 6
Safari X
```

**Comment by ArthurGoupil** [comment]
Thanks @ai but the thing here is that Safari is not mentioned in the user agent for some versions of `AppleMail` (like the one provided in the first message), and we're using [browserslist-useragent-regexp](https://github.com/browserslist/browserslist-useragent-regexp) to redirect users to a warning page. But I guess there is no solution then.
